src/EditorFeatures/Core/Shared/Extensions/TelemetryExtensions.cs

```diff
@@ -40,7 +40,9 @@ public static short GetScopeIdForTelemetry(this FixAllScope scope)
                 FixAllScope.Document => 1,
                 FixAllScope.Project => 2,
                 FixAllScope.Solution => 3,
-                _ => 4,
+                FixAllScope.ContainingMember => 4,
```

Comment:
We previously reported "Custom" scopes as `4` but will now report them as `6` with this change right? I assume we want to keep the telemetry reporting the same for custom scopes

Reply:
Ah, thanks good catch. I'll make the change.